src/ert/run_models/event.py

```diff
@@ -26,11 +27,17 @@ class RunModelUpdateBeginEvent(RunModelEvent):
 
 
 @dataclass
-class RunModelUpdateEndEvent(RunModelEvent):
-    smoother_snapshot: Optional[SmootherSnapshot] = None
+class RunModelCSVEvent(RunModelEvent):
+    name: str
+    header: List[str]
+    data: Sequence[Sequence[Union[str, float]]]
+
+
+@dataclass
+class RunModelUpdateEndEvent(RunModelCSVEvent):
+    extra: Dict[str, str]
 
```

Comment:
Same as AnalysisErrorEvent

Reply:
Same logic as above, I could just use the same logic for all three data event types.